learning_service/Taskfile.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+
+vars:
+  BUILDER_IMAGE: learning_service_image
+
+tasks:
+  prepare:
+    - task: venv_create
+    - task: venv_activate
+    - task: pip_install
+  venv_create:
+    cmds:
+      - python3 -m venv venv
+  venv_activate:
+    cmds:
+      - unameOut=tolower("$(uname -s)")
+      - if [ unameOut == "msys" ] || [ unameOut == "cygwin" ]; then source venv/Scripts/activate; else source venv/bin/activate fi
+  pip_install:
+    cmds:
+      - pip install -r requirements.txt
+  build_docker:
+    deps: [prepare]
+    cmds: ["docker build -t {{.BUILDER_IMAGE}} ."]
+  docker_run:
+    deps: [build_builder]
```

Comment:
There is no `build_builder` task anywhere - the `build_docker` seems to be doing that, but it's named as if it was supposed to build the actual application's image, not the builder one.